yt_dlp/postprocessor/xattrpp.py

```diff
@@ -29,12 +29,21 @@ class XAttrMetadataPP(PostProcessor):
         'user.dublincore.date': 'upload_date',
         'user.dublincore.contributor': 'uploader',
         'user.dublincore.format': 'format',
+        'com.apple.metadata:kMDItemWhereFroms': 'webpage_url',
         # We do this last because it may get us close to the xattr limits
         # (e.g., 4kB on ext4), and we don't want to have the other ones fail
         'user.dublincore.description': 'description',
         # 'user.xdg.comment': 'description',
```

Comment:
maybe `com.apple.metadata:kMDItemWhereFroms` should go after `description` since even without any url is already longer than many videos' description strings?